open-beta.md

```diff
@@ -0,0 +1,28 @@
+# VANet 2 Open Beta
+
+As of May 2, 2021 the VANet v2 website is in public beta. This means that while we are still expecting some kinks, we need the community's help to find any further issues.
+
+VANet v2 includes the following enhancements.
+
+-   New Authentication System
+-   Reworked website
+-   Major backend rewrite
+-   Support tickets
+-   Session & account management
+-   Paid tier for users removed (not airlines)
+-   Public browsing of airport/aircraft data
+-   New developer portal and API Key system
+
+Please note that VANet v2 is only designed to be a rework of VANet v1 with a small amount of new features. All the cool things we've announced over the past few months will come in future updates - this update just sets the stage for that.
+
+## How do I participate?
+
+To participate, simply sign up at <https://preview.vanet.app> and use the website as you would normally. You do not need to use the same authentication provider as VANet v1. By signing up, on this site we will assume you are opting into the beta and may contact you with questions or further information.
```

Comment:
Change comma from 'By signing up,'

Correction:

By signing up on this site, we will assume you are opting into the beta...